src/mod_ping.erl

```diff
@@ -232,9 +224,37 @@ user_keep_alive(Acc, JID) ->
     start_ping(JID#jid.lserver, JID),
     Acc.
 
+user_ping_timeout(Acc, _JID) -> % TODO is this hook really needed?
```

Comment:
This module doesn't need it in the current state but some custom extensions might rely on it.